api/sshuser_test.go

```diff
@@ -320,7 +320,7 @@ func TestPostUserSSHCertificate(t *testing.T) {
 			KeyUsages:      sshkeyUsage,
 			maxValidity:    map[string]uint64{config.SSHUserCertEndpoint: 3600},
 			validity:       3600,
-			KeyMeta:        &proto.KeyMeta{Identifier: "sshuserid1"},
+			KeyMeta:        &proto.KeyMeta{Identifier: "sshuserid"},
```

Comment:
The test case `requestCancelled` aims to cancel the request by context. The correct identifier should be "sshuserid" instead of "sshuserid1". Using "sshuserid1" would make the target function fail before the context timeout.